src/main/support.js

```diff
@@ -75,8 +75,8 @@ function getSupportInfo(version, opts) {
     .map(language => {
       let parsers;
       // Prevent breaking changes
-      if (language.name === "Markdown" || language.name === "TypeScript") {
-        parsers = [language.name.toLowerCase()];
+      if (language.name === "Markdown") {
+        parsers = ["markdown"];
```

Comment:
Merge mistake?

Reply:
Intended. There is no need in this overwriting for TS.